.github/workflows/release.yml

```diff
@@ -511,22 +511,73 @@ jobs:
 
   trigger-docs-update:
     if: ${{ inputs.skip-release-notes != true }}
-    needs: push-tags
+    needs:
+      - push-tags
```

Comment:
lets maybe use array like [] https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds